cmd/medius/images/push.go

```diff
@@ -300,7 +300,7 @@ func (b *buildAndPublish) rebuildNeeded(entry *common.Entry) (bool, error) {
 		if err != nil {
 			return false, err
 		}
-		if imageSha != artifactInfo.SHA256Sum {
```

Comment:
Maybe in a follow up, but `imageSha` and `b.getImageSha` should be adapted too.

Reply:
I've changed it in this PR. I think it is related with this.